### PR TITLE
Circleci project setup

### LINKED ISSUE #9: Setup CircleCI integration & auto-deployment for BE
- [ ] DB migrations happen automatically

### PR DESCRIPTION
**Pull request by jobannon**
### Sidebar Checklist
​
- [x] Ran test suite
- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project
- [ ] Added to Readme
​
### Issues Resolved
Resolves #9 
​
### Problem Addressed We needed CI for our project on the back end.  Circle CI selected and implemented.
​
### Solution Implemented.  Circle CI implemented 
​
### Screenshots (optional)
​
image

### CODE REVIEW

**Comment by DanielEFrampton** [comment]
@rosetta-team/rosetta  This is a good candidate for using squash-and-merge instead of rebase and merge.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      - run: 
+          # name: setup_exports
+          # command: |
+          #     export FLASK_APP=flaskr/app 
+          #     export FLASK_ENV=development 
+          # # command: |
+          # #     echo 'export FLASK_APP=flaskr/app' >> $BASH_ENV
+          # #     echo 'export FLASK_ENV=development' >> $BASH_ENV
+      # - run:
+          # name: Test
```
Comment (DanielEFrampton) [inline .circleci/config.yml:23]: Let's take all these commented lines out

File: .python-version [hunk 2]
```diff
@@ -0,0 +1 @@
+3.7.7
```
Comment (DanielEFrampton) [inline .python-version:1]: This file got created on my machine at some point as well but I noticed it has never been pushed; does anyone know where it comes from, and whether we need it?

Comment (mintona) [reply .python-version:1]: I also had it at some point but now I don't see it!

File: .circleci/config.yml (hunk 1 above)
Comment (jobannon) [reply .circleci/config.yml]: Good Catch!  Removed!

**Pull request merged**